.vscode/tasks.json

```diff
@@ -62,50 +85,116 @@
             "group": {
                 "kind": "build",
                 "isDefault": true
             },
             "problemMatcher": []
+        },
+        {
+            "label": "Load app on device (Linux)",
+            "type": "shell",
+            // Copies the ledger udev rule file to the /etc/udev/rules.d/ directory if it does not exist, then reloads the rules and triggers udev. After that, it executes make load in the container to load the app on a physical device.
+            "command": "if [ ! -f '/etc/udev/rules.d/${config:linux_udev_ledgerblue_rule_file}' ]; then sudo cp .vscode/${config:linux_udev_ledgerblue_rule_file} /etc/udev/rules.d/ && sudo udevadm control --reload-rules && sudo udevadm trigger; fi && docker exec -it ${config:container_name} bash -c 'export BOLOS_SDK=${input:sdk} && make load'",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            },
+            "problemMatcher": []
+        },
+        {
+            "label": "Load app on device (macOS)",
+            "type": "shell",
+            // Checks that virtual env is installed, otherwise installs it. Then installs ledgerblue in a virtualenv and side loads the app APDU file.
+            "command": "cmd=\"[ -d 'ledger' ] || python3 -m virtualenv ledger && source ledger/bin/activate && python3 -m pip install ledgerblue && python3 -m ledgerblue.runScript --scp --fileName bin/app.apdu --elfFile bin/app.elf\"; if python3 -m virtualenv --version >/dev/null 2>&1; then eval \"${cmd}\"; else python3 -m pip install virtualenv && eval \"${cmd}\"; fi",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            },
+            "problemMatcher": []
+        },
+        {
+            "label": "Load app on device (Windows with PowerShell)",
+            "type": "shell",
+            // Checks that virtual env is installed, otherwise installs it. Then installs ledgerblue in a virtualenv and side loads the app APDU file.
+            "command": "cmd.exe /C 'if exist ledger (call ledger\\Scripts\\activate.bat && python -m ledgerblue.runScript --scp --fileName bin/app.apdu --elfFile bin/app.elf) else (python -m pip install virtualenv && python -m venv ledger && call ledger\\Scripts\\activate.bat && python -m pip install ledgerblue && python -m ledgerblue.runScript --scp --fileName bin/app.apdu --elfFile bin/app.elf)'",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            },
+            "problemMatcher": []
+        },
+    ],
+    "inputs": [
+        {
+            "id" : "sdk",
+            "type": "pickString",
+            "description": "Choose a SDK to build with",
+            "options": [
+                "/opt/nanos-secure-sdk",
+                "/opt/nanox-secure-sdk",
+                "/opt/nanosplus-secure-sdk",
+                "/opt/stax-secure-sdk",
```

Comment:
Could we use docker env ariable instead? e.g. https://github.com/LedgerHQ/ledger-app-builder/blob/master/lite/Dockerfile#L49 for nanos

Reply:
Ok, by doing `BOLOS_SDK=$(echo ${input:sdk})` I can have variable expansion of a variable defined inside the docker container. I updated the tasks.json file.